meeteval/wer/__main__.py

```diff
@@ -525,6 +553,28 @@ def add_argument(self, command_parser, name, p):
                      '- lower,rm(.?!,): Lowercase the transcript and remove punctuations (.,?!).',
                 choices=[None, 'lower,rm(.?!,)'],
             )
+        elif name == 'distancetype':
+            command_parser.add_argument(
+                '--distancetype',
+                choices=['1', '2', '21'],
+                help='Specifies the costs used for substitutions in the greedy distance calculation.\n'
+                     'Choices:\n'
+                     '- 1: A substitution cost of 1 (standard Levenshtein distance)\n'
+                     '- 2: A substitution cost of 2 (subst = ins + del)\n'
+                     '- 21: Optimize with 2 until convergence, then optimize with 1 until convergence. This is the '
+                     'default and recommended for best results.'
+            )
+        elif name == 'assignment_initialization':
```

Comment:
Have you tried the alternatives? Is there a motivation to provide them?

random: We should not allow an option that yields non deterministic results

Reply:
I haven't tried them on large dataset, but even the constant initialization yielded good results for the examples I looked at. The motivation is that the ORC-WER does not need speaker labels, but the cp initialization needs them.

Reply:
An unbalanced initial assignment could have side effects.
But maybe the side effect is only, that the code takes some more iterations.

Has ORC-WER issues with cp? I'd like to have always the same init and in general it is best,
when nobody changes arguments to increase the probability, that people can resonably compare across publications (i.e. we don't want to have the asclite issue, where everyone has a slightly different evaluation).

Reply:
I removed the option. Now, cp is always used.